src/classiceditorui.js

```diff
@@ -53,6 +54,14 @@ export default class ClassicEditorUI {
 		 */
 		this.focusTracker = new FocusTracker();
 
+		/**
+		 * Instance of the {@link module:core/keystrokehandler~KeystrokeHandler}.
```

Comment:
Could be explained what this particual instance is supposed to control compared to the editor keystrokes.